### PR TITLE
Added udev example

### PR DESCRIPTION
**Pull request by kayrus**
/cc @robszumski 
Rob, can you suggest another udev example?

### CODE REVIEW

**Comment by robszumski** [comment]
I don't know a ton about udev rules so I found this entire doc hard to follow. Adding some comments inline.

File: os/using-systemd-and-udev-rules.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Using systemd and udev Rules 
+
+In our example we will use libvirt VM. First of all we have to create systemd unit file `/etc/systemd/system/test.service`:
```
Comment (robszumski) [inline os/using-systemd-and-udev-rules.md]: Libvirt is one of our less popular platforms. Is there a way we can make this guide generic or use a different platform as the example?

Comment (robszumski) [inline os/using-systemd-and-udev-rules.md]: Why are we creating this unit? What does it do?

File: os/using-systemd-and-udev-rules.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+```
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/echo 'device has been attached'
+```
+
+Then we have to start `udevadm monitor --environment` to monitor kernel events.
+
+Once you've attached virtio libvirt device (i.e. `virsh attach-disk coreos /dev/VG/test vdc`) you'll see similar udev output:
```
Comment (robszumski) [inline os/using-systemd-and-udev-rules.md]: Where do I see this output?

File: os/using-systemd-and-udev-rules.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+MAJOR=254
+MINOR=16
+SEQNUM=1327
+SUBSYSTEM=block
+SYSTEMD_WANTS=test.service
+TAGS=:systemd:
+USEC_INITIALIZED=545954447
+```
+
+Accoring to text beyond udev rule should look this way:
```
Comment (robszumski) [inline os/using-systemd-and-udev-rules.md]: "according"

Comment (robszumski) [inline os/using-systemd-and-udev-rules.md]: According to what text?

File: os/using-systemd-and-udev-rules.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+```
+ACTION=="add", SUBSYSTEM=="block", TAG+="systemd", ENV{SYSTEMD_WANTS}="test.service"
+```
+
+Now when we use this command `virsh attach-disk coreos /dev/VG/test vdc` on host machine, we should see `device has been attached` message inside CoreOS node.
```
Comment (robszumski) [inline os/using-systemd-and-udev-rules.md]: So the test.service unit triggers the disk to be attached?

File: os/using-systemd-and-udev-rules.md (hunk 1 above)
Comment (robszumski) [inline os/using-systemd-and-udev-rules.md]: Can we rename this unit to something like `disk-notify` or `device-attach` or `device-notify`?

File: os/using-systemd-and-udev-rules.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+## Cloud-Config example
+
+Cloud-Config example should look this way:
```
Comment (robszumski) [inline os/using-systemd-and-udev-rules.md]: "To use the unit and udev rule with cloud-config, modify this example as needed:"

File: os/using-systemd-and-udev-rules.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+        Description=Notify about attached device
+
+        [Service]
+        Type=oneshot
+        ExecStart=/usr/bin/echo 'device has been attached'
+```
+
+## Another Examples
+
+For another real examples, check out these documents:
```
Comment (robszumski) [inline os/using-systemd-and-udev-rules.md]: How do these relate?

**Comment by kayrus** [comment]
Updated

**Comment by robszumski** [comment]
Let's just get this in. LGTM and we can tweak if needed.
